openSpotServer/parkingSpot/parkingSpotMask.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import cv2
+
+def get_mask_of_parking_spot(original_image, parking_spot_polygon, overlay=False):
+	""" 
+	The function returns a masked parking spot.
+
+	Parameters: 
+	original_image : Original grayscale image.
+						gray = cv2.imread('car_output.jpg', 0) can be used to get grayscale 0 Flag stands for grayscale.
+						gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY) can be used to get grayscale
+	parking_spot_polygon : np.array that depicts the parking spot
+	overlay : True returns an overlay on original image
+			  False returns a mask array,
+			  	if [i][j] is 1 that means that pixel is part of a parking spot
+			  	if [i][j] is 0 that means that pixel is not part of a parking spot
+
+	Returns: 
+	output_image: returns an overlay of parking spot on original image or 
+				  a mask array where:
+			  		if [i][j] is 1 that means that pixel is part of a parking spot
+			  		if [i][j] is 0 that means that pixel is not part of a parking spot
+	"""
+	output_image = np.zeros(shape = original_image.shape, dtype = "uint8")
+	if overlay:
+		output_image = original_image
+
+	cv2.fillPoly(output_image, [parking_spot_polygon], color = (1))
+	return output_image
```

Comment:
Does this have two possible returns? 
A mask array or an image?

If so can we split his out into two separate functions Its confusing to have two drastically different output types from the same function. 

something like `get_mask_of_parking_spots()`
and `get_overlay_of_parking_spots()`

You can call `get mask_of_parking_spots` within `get_overlay_of_parking_spots`

Reply:
The return is the same thing however if you try to draw the picture when overlay is false, it will be hard to see the mask of the spot since background is of value 0 and spot is value 1. (max 255)

Reply:
I separated into 2 different functions